misc/tools/install.sh

```diff
@@ -116,7 +128,7 @@ done
 echo
 if ! (php -m | grep -q pdo_mysql) ; then
 	echo "php module pdo_mysql has not been found"
-	echo "try 'sudo apt-get install php5-mysqlnd' on Debian/Ubuntu based systems"
+	echo "try 'sudo apt-get install php7.0-mysql' on Debian/Ubuntu based systems"
```

Comment:
Ist php7-mysql als alias möglich?

Reply:
So in der Art:?

`# php package required`
`php_package=php7.0-mysql`
`.`
`.`
`.`
`echo "try 'sudo apt-get install $php_package' on Debian/Ubuntu based systems"`